packages/react-components/src/checkbox/src/Checkbox.css

```diff
@@ -1,4 +1,6 @@
 .o-ui-checkbox {
+    --o-ui-focus-ring-stroke-width: 2px;
+    --o-ui-indeterminate-stroke-width-md: 5px;
```

Comment:
Why `-md` suffix for the for the indeterminate but not for `--o-ui-focus-ring-stroke-width`. Don't care if we add it or remove it but if it should be consistant if there isn't any good reason not to be.